src/docs/taiko.mdx

```diff
@@ -0,0 +1,165 @@
+    
+    <Parameter name="Proposer and Prover Implementations" value={<Reference label="taiko-client" url="https://github.com/taikoxyz/taiko-client" />} tooltip="The different client implementations of the rollup" />
+
+    <Parameter name="Transaction Types" tooltip="The types of transactions supported on the Rollup" />
+
+        - **Type 0** - User Transaction. Represent [pre-EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) transactions
+        - **Type 1** - User Transaction. Represent legacy transactions post [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) transactions
+        - **Type 2** - User Transaction. [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) transactions
+    ___
+
+    <MultiRowParameters title="Messaging" tooltip="The Rollup provides native communication with L1. Anyone is able to transmit arbitrary messages and the Taiko protocol will provide proof of the data transfer on the destination chain" data={[
+        {
+            title: 'L1 → L2',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: 'The maximum time it takes for a message to be made available on the rollup after it is included in the Ethereum block.',
+                    value: '90 minutes',
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of purely sending a message from Ethereum to Тaiko. In case of automaitic claiming on L2 Senders on L1 pay an additional ETH fee determined by market conditions on Taiko.',
+                    value: '~110 000 L1 gas + ~440 000 L2 gas (in case of manual delivery)',
+                },
+                {
+                    label: 'Delivery method',
+                    tooltip: 'How the message on the target chain is delivered. In both automatic and manual delivery, users are waiting for a prover to generate validity proof for the target chain (max 90 minutes). In case of manual delivery users need to claim the message themselves as opposed to automatic',
+                    value: 'Both manual and automatic delivery are supported'
+                },
+            ],
+        },
+        {
+            title: 'L2 → L1',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: 'The maximum time it takes for a message to be made available on Ethereum after being included in an Taiko block and proved for L1.',
+                    value: '90 minutes',
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of purely sending a message from Taiko to Ethereum. In case of automaitic claiming on L1 Senders on L2 pay an additional ETH fee determined by market conditions on Ethereum.',
```

Comment:
Is there automatic delivery for L2 to L1 communication? 
Does it really cost 440k gas for purely receiving the message? Sounds a bit too much, maybe you've looked at a big message with lots of calldata?

Reply:
It's 440k for merkle inclusion proof of the message